current/runtime/storage.md

```diff
@@ -4,22 +4,123 @@ lang: en
 title: Runtime Storage
 ---
 
-Runtime storage allows you to store data in your blockchain which can be accessed from your runtime logic and persists between blocks.
+Runtime storage allows you to store data in your blockchain that is persisted between blocks and can be accessed from
+within your runtime logic. Storage should be one of the most critical concerns of a blockchain runtime developer. This
+statement is somewhat self-evident, since one of the primary objectives of a blockchain is to provide decentralized
+consensus about the state of the underlying storage. Furthermore, well designed storage systems reduce the load on
+nodes in the network, which will lower the overhead for participants in your blockchain. Substrate exposes a set of
+layered, modular storage APIs that allow runtime developers to make the storage decisions that suit them best. However,
+the fundamental principal of blockchain runtime storage is to minimize its use. This document is intended to provide
```

Comment:
I think this should be principle by definition 1c
https://www.merriam-webster.com/dictionary/principle